google/cloud/bigtable/examples/bigtable_grpc_credentials.cc

```diff
@@ -152,18 +129,50 @@ void GCECredentials(int argc, char* argv[]) {
     }
   }
   //! [test gce credentials]
-  (std::move(project_id), std::move(instance_id));
+  (argv[0], argv[1]);
+}
+
+void RunAll(std::vector<std::string> argv) {
+  if (!argv.empty()) throw Usage{"auto"};
+
+  auto const project_id =
```

Comment:
Opt Nit: Might it be clearer to run the checking `for` loop first, and then make these just `*GetEnv()` calls?

Reply:
Done.